src/app/home/sponsors/sponsors.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../../styles-mat-variables";
+
+.sponsor {
+  margin: 40px;
+}
+
+.sponsors-logo {
+  max-width: 80%;
+  max-height: 40%;
+
+  @media #{$medium-and-up} {
+    max-width: 100%;
+  }
+
+  .a_button {
```

Comment:
This notation does not follow css standards and is used only once, maybe it should be an id. If possible use a more descriptive name, like: become-sponsor-button